data/language/en-US.txt

```diff
@@ -240,6 +240,7 @@ STR_6212    :Gray sandstone
 STR_6274    :Can't set color scheme...
 STR_6307    :Color scheme: {BLACK}{STRINGID}
 STR_6328    :{SMALLFONT}{BLACK}With this option enabled, giant screenshots will have a transparent background instead of the default black color.
+STR_6333    :Nearest Neighbor
```

Comment:
It's not too much of a problem since you're going to remove it anyway, but we *never* update language files from this PR (except en-GB). Those are updated in the Localisation repository and updated nightly.